USE-CASES/security-categories.csv

```diff
@@ -1,76 +1,76 @@
-2.1.3Irrigation in Outdoor Environment,no,conf,no,yes,"Privacy->Confidential"
-2.1.4Automatic milking system for dairy farm,no,no,no,yes,"Safety for people; not cows?  Also, cameras assumed not to include people"
-2.1.5Pest control in Open-field,no,yes,yes,yes,"Drones can be airspace risk and privacy risk (cameras)"
-2.1.6Livestock Health Management,no,no,maybe,yes,"Diseases can cross to people (bird flue); cameras can be set to monitor livestock only"
-2.1.7Agricultural Machinery Management,no,no,yes,yes,"Safety issue if dangerous maintenance problem hidden"
-2.2Smart City,,,,,
-2.2.1Smart City Geolocation,,,,,
-2.2.2Smart City Dashboard,,,,,
-2.2.3Interactive Public Spaces,,,,,
-2.2.4Meeting Room Event Assistance,,,,,
-2.2.5Cross-Domain Discovery in a Smart Campus,,,,,
-2.2.6Cultural Spaces (Museums),,,,,
-2.3Building Technologies,,,,,
-2.3.1Smart Building,,,,,
-2.3.2Connected Building Energy Efficiency,,,,,
-2.3.3Automated Smart Building Management,,,,,
-2.3.4Portable Building Applications,,,,,
-2.4Manufacturing,,,,,
-2.4.1Production Monitoring,,,,,
-2.4.2Cross-protocol Interaction in Industry 4.0 Scenarios,,,,,
-2.5Retail,,,,,
-2.5.1Retail Operations,,,,,
-2.5.2Retail All Stop Button (Outdoor Emergency Stop Plunger),,,,,
-2.5.3Retail Indoor Door Sensor,,,,,
-2.5.4Retail Indoor and Outdoor Freezers,,,,,
-2.5.5Retail Kitchen Refrigerator,,,,,
-2.5.6Retail Restroom Devices,,,,,
-2.5.7Retail Lighting Control,,,,,
-2.5.8Retail Outdoor Canopy Lighting Control,,,,,
-2.5.9Retail Fountain Drink Ice Machine,,,,,
-2.5.10Retail Camera Device,,,,,
-2.6Health,,,,,
-2.6.1Public Health,,,,,
-2.6.1.1Smart City Public Health Monitoring,,,,,
-2.6.1.2Interconnected Medical Devices in a Hospital ICU,,,,,
-2.6.2Private Health,,,,,
-2.6.2.1Health Notifiers,,,,,
-2.6.3Biomedical Devices,,,,,
-2.6.3.1Digital Microscopes,,,,,
-2.7Energy,,,,,
-2.7.1Smart Grids,,,,,
-2.8Transportation,,,,,
-2.9Automotive,,,,,
-2.9.1Smart Car Configuration Management,,,,,
-2.10Smart Home,,,,,
-2.10.1Audio/Video,,,,,
-2.10.1.1Media Use Cases,,,,,
-2.10.2Home WoT devices synchronize to TV programs,,,,,
-2.10.3Leaving and Coming Home,,,,,
-2.11Education,,,,,
-2.11.1Education Shared Devices,,,,,
-3.Use Cases for multiple domains,,,,,
-3.1Discovery,,,,,
-3.2Multi-Vendor System Integration - Out of the box interoperability,,,,,
-3.3Virtual Thing,,,,,
-3.4Digital Twin,,,,,
-3.4.1Digital Twin (1),,,,,
-3.4.2Digital Twin (2),,,,,
-3.5Cross Protocol Interworking,,,,,
-3.6Multimodal System Integration,,,,,
-3.6.1Multimodal Recognition Support,,,,,
-3.6.2Enhancement of Synergistic Interactions,,,,,
-3.7Accessibility,,,,,
-3.7.1Audiovisual Devices Acting as Smartphone Extensions,,,,,
-3.8Security,,,,,
-3.8.1OAuth2 Flows,,,,,
-3.9Lifecycle,,,,,
-3.9.1Device Lifecycle,,,,,
-3.10VR/AR,,,,,
-3.10.1AR Virtual Guide,,,,,
-3.11Edge Computing,,,,,
+"UC","Public Service","Private/PII","Business Confidential","Safety Critical","Business Critical","Comment"

```

Comment:
Add a column to record whether the categorization has been confirmed by UC authors.
```suggestion
"UC","Public Service","Private/PII","Business Confidential","Safety Critical","Business Critical","Comment","Validated?"

```

Reply:
Don't commit suggest yet, I need to update to add commas to the end of all the rows...